free-1.md

```diff
@@ -45,6 +59,8 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There's probably tons of DSLs in this domain. Physical simulations are very important in industry and acadamia, so I wouldn't be surprised if there were hundreds of various programs out on the internet. [This page](https://phet.colorado.edu/en/simulations/category/physics) alone has a massive number of standalone, free, interactive simulations (that aren't programmable). However, like I said earlier, a lot of the simple applets used in Physics classes are written in Java, which can make them hard to run. Professional simulation packages may be expensive or suffer from feature bloat (that is, it is hard to use them). [This software](http://www.comsol.com/products) looks like a good example as it supports many different kinds of simulation, but you have to contact sales just to get an idea what the price is. The proposed setup would hopefully be pretty straightforward and simple to work with.
```

Comment:
This is important - it's another important consideration for this project that you make sure to distinguish yourself from these. I think you've set up a good basis for doing this, and if you pick this project you want to focus on what advantages a DSL mindset can offer over these existing tools.